homeassistant/components/insteon/cover.py

```diff
@@ -19,33 +21,18 @@
 
 async def async_setup_platform(hass, config, async_add_entities, discovery_info=None):
     """Set up the Insteon platform."""
-    if not discovery_info:
-        return
-
-    insteon_modem = hass.data["insteon"].get("modem")
-
-    address = discovery_info["address"]
-    device = insteon_modem.devices[address]
-    state_key = discovery_info["state_key"]
-
-    _LOGGER.debug(
-        "Adding device %s entity %s to Cover platform",
-        device.address.hex,
-        device.states[state_key].name,
+    async_add_insteon_entities(
+        hass, DOMAIN, InsteonCoverDevice, async_add_entities, discovery_info
     )
 
-    new_entity = InsteonCoverEntity(device, state_key)
-
-    async_add_entities([new_entity])
-
 
-class InsteonCoverEntity(InsteonEntity, CoverEntity):
+class InsteonCoverDevice(InsteonEntity, CoverEntity):
```

Comment:
Please name entity classes SomethingEntity.

Reply:
I did this for all the entity classes. Thanks.